src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -68,10 +68,8 @@ static function ($key, $value, $isHit) {
      *
      * @param string                 $file         The PHP file were values are cached
      * @param CacheItemPoolInterface $fallbackPool A pool to fallback on when an item is not hit
-     *
-     * @return CacheItemPoolInterface
      */
-    public static function create(string $file, CacheItemPoolInterface $fallbackPool)
+    public static function create(string $file, CacheItemPoolInterface $fallbackPool): CacheItemPoolInterface
```

Comment:
```suggestion
    public static function create(string $file, CacheItemPoolInterface $fallbackPool): static
```

I don't quite understand why the previously annotated return type was so wide, tbh.